tests/PHPStan/Rules/Functions/CallToFunctionParametersRuleTest.php

```diff
@@ -1388,4 +1388,22 @@ public function testFlockParams(): void
 		]);
 	}
 
+	public function testJsonEncodeDecodeParams(): void
+	{
+		$this->analyse([__DIR__ . '/data/json_encode_decode.php'], [
```

Comment:
I named this test encode/decode because initially I had both cases covered here.

turns out though, that because of json_encode() accepts so many JSON flags, we should do this in a separate followup PR, because DX suffers from the huge int-mask generated from it.

lets discuss it in the followup then

Reply:
Right now it's just "decode" so name it decode please.